plugins/Live/Model.php

```diff
@@ -509,4 +531,25 @@ private function getWhereClauseAndBind($whereClause, $bindIdSites, $idSite, $per
         }
         return array($whereBind, $where);
     }
+
+    /**
+     * @param $ecommerceIdGoal
+     * @return string
+     */
+    private function getSqlEcommerceConversionsLifeTimeMetricsForIdGoal($ecommerceIdGoal)
+    {
+        $sql = "SELECT
+                    COALESCE(SUM(" . LogAggregator::getSqlRevenue('revenue') . "), 0) as lifeTimeRevenue,
+                    COUNT(*) as lifeTimeConversions,
+                    COALESCE(SUM(" . LogAggregator::getSqlRevenue('items') . "), 0)  as lifeTimeEcommerceItems
+					FROM  " . Common::prefixTable('log_visit') . " AS log_visit
+					    LEFT JOIN " . Common::prefixTable('log_visit') . " AS log_visit_visitors
+					    ON (log_visit.idsite = log_visit_visitors.idsite AND log_visit.idvisitor = log_visit_visitors.idvisitor)
```

Comment:
wouldn't we safe one join when matching by idvisitor directly? do we not have the idVisitor in Live API?